devtool/client/src/components/RandomDataGenerator.tsx

```
@@ -0,0 +1,256 @@
+                            disabled={state.loading}
+                        />
+                    </label>
+                </div>
+
+                <div className="devtools-button-group">
+                    <button
+                        className="devtools-button devtools-button-primary"
+                        onClick={handleGenerate}
+                        disabled={state.loading}
+                    >
+                        {state.loading ? '⏳ Generating...' : '🎲 Generate Data'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)